lib/unitary_calculator_basic.h

```diff
@@ -15,8 +15,10 @@
 #ifndef UNITARY_CALCULATOR_BASIC_H_
 #define UNITARY_CALCULATOR_BASIC_H_
 
+
+#include <algorithm>
+#include <complex>
```

Comment:
Why add these ?

Reply:
Thanks, that's a good point. I'll remove that in the next PR when I add support for multi-qubit gates.